packages/unity-bootstrap-theme/stories/molecules/cookie-consent/cookie-consent.stories.js

```diff
@@ -2,14 +2,27 @@ import React, { useEffect } from "react";
 import { allCookieConsentJS } from "../../../src/js/cookie-consent";
 import "../../../src/css/cookie-consent.css";
 
-import { fullLayoutDecorator } from "../../../../../shared/components/Layout";
-
 export default {
   title: "Molecules/Cookie Consent",
-  decorators: [ fullLayoutDecorator ],
```

Comment:
since we are using this in 2 places move decorator to its own file and import it, maybe put it here:`packages/unity-bootstrap-theme/.storybook/decorators.js` or if we think we will start using it with other packages we can put it in the shared folder

Reply:
If they are in the same file, no need to create a new file at this point